src/Core/Settings/GlobalSettings.cs

```diff
@@ -521,7 +522,7 @@ public class SyslogSettings
         /// The absolute path to a Certificate (DER or Base64 encoded with private key).
         /// </summary>
         /// <remarks>
-        /// The certificate path and <see cref="CertificatePassword"/> are passed into the <see cref="System.Security.Cryptography.X509Certificates.X509Certificate2.X509Certificate2(string, string)" />.
+        /// The certificate path and <see cref="CertificatePassword"/> are passed into the <see cref="X509Certificate2" />.
```

Comment:
❓ Unrelated change?

Reply:
This was an error in rider that was bugging me so I added it 🤷